### PR TITLE
Tag or Commit Hash ID Links

### PR DESCRIPTION
**Pull request by dkosteck**
Updates to use tags and commit hashes as links to spec on GitHub, without the need for UUID or mapping. As long as semantic versioning of tags is observed, and the code is run from the repo, this should allow for linking to specific versions of tests/test suites.

### CODE REVIEW

**Comment by dkosteck** [comment]
This approach has been deemed desirable, so reviews are now encouraged

**Review by jianzzha — APPROVED** [review]
Looks good to me.

**Comment by akiselev1** [comment]
We may consider a TL;DR version of tagging in the future doc version. A new contributor will tell us :-)
Is there a new release tag for this PR?

**Comment by dkosteck** [comment]
> We may consider a TL;DR version of tagging in the future doc version. A new contributor will tell us :-) Is there a new release tag for this PR?

@akiselev1 So as far as a release tag for this PR, that's a good question. So far the releases were mostly being used by me for testing this functionality, but if we think once this is merged to main that the repo is in a "steady" state, then perhaps we create the first real release and give it a real version number. Thoughts @jianzzha @akiselev1 ?

**Comment by akiselev1** [comment]
I think this may be a good place to demo this new functionality at least with a minor release increment.

**Review by akiselev1 — COMMENTED** [review]
We should highlight that tagging will be handled by the repo maintainers

**Review by akiselev1 — APPROVED** [review]

**Pull request merged**
